ruby/all-but-the-first-element-from-array.md

```diff
@@ -21,6 +21,34 @@ rest_of_list = list
 # => [2,3,4]
 ```
 
+## Parallel assignment
+
+We can also use parallel assignment to get the first element of the list. This
+does not mutate the original array.
+
+```ruby
+head, *tail = list
+#=> [1,2,3,4]
+head
+#=> 1
+tail
+#=> [2,3,4]
+
+list # is not modified
+#=> [1,2,3,4]
+```
+
+When using parallel assignment on an empty array, the tail will not be `nil`.
```

Comment:
It's completely fine with me. I've moved the example down, and I now mention `#drop` again in the section about empty arrays.